tests/integration/changed-yum-conf/test_patch_yum_conf.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+def test_yum_patch(convert2rhel, shell):
```

Comment:
```suggestion
def test_yum_patch(convert2rhel, shell):
    """Test the scenario in which the user modifies /etc/yum.conf before the conversion.

    In that case during the conversion the config file does not get replaced with the config file from the RHEL package
    (%config(noreplace)) and we need to make sure that we patch the config file to get rid of the distroverpkg config
     key. Leaving the distroverpkg key there would lead to errors when calling yum after the conversion for not
    expanding the $releasever variable properly.
    """
```